modules/report-server.nix

```diff
@@ -25,7 +25,7 @@ in {
       groups.report-server = {};
     };
 
-    networking.firewall.allowedTCPPorts = [ cfg.port ];
+    networking.firewall.enabled = false;
```

Comment:
Actually for report-server, I'd just open up the port. I'm confident it will work.